File: docs/components_page/components/card/color.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

card_content <- list(
  dbcCardHeader("Card Header"),
  dbcCardBody(
    list(
      htmlH5("Card Title", className = "card-title"),
      htmlP(
        "This is some card content that we'll reuse",
        className = "card-text"
      )
    )
  )
)

cards <- htmlDiv(
  list(
    dbcRow(
      list(
        dbcCol(dbcCard(card_content, color = "primary", inverse = TRUE)),
        dbcCol(dbcCard(card_content, color = "secondary", inverse = TRUE)),
        dbcCard(card_content, color = "info", inverse = TRUE)
      ),
      className = "mb-4"
    ),
    dbcRow(
      list(
        dbcCol(dbcCard(card_content, color = "success", inverse = TRUE)),
        dbcCol(dbcCard(card_content, color = "warning", inverse = TRUE)),
        dbcCard(card_content, color = "danger", inverse = TRUE)
      ),
      className = "mb-4"
    ),
    dbcRow(
      list(
        dbcCol(dbcCard(card_content, color = "light")),
        dbcCol(dbcCard(card_content, color = "dark")),
        dbcCard(card_content, color = "danger", inverse = TRUE)
      )
    )
  )
)

File: docs/components_page/components/card/color.jl
using DashBootstrapComponents

card_content = [
    dbc_cardheader("Card header"),
    dbc_cardbody([
        html_h5("Card title", className = "card-title"),
        html_p("This is some card content that we'll reuse", className = "card-text"),
    ]),
];

cards = html_div([
    dbc_row(
        [
            dbc_col(dbc_card(card_content, color = "primary", inverse = true)),
            dbc_col(dbc_card(card_content, color = "secondary", inverse = true)),
            dbc_col(dbc_card(card_content, color = "info", inverse = true)),
        ],
        className = "mb-4",
    ),
    dbc_row(
        [
            dbc_col(dbc_card(card_content, color = "success", inverse = true)),
            dbc_col(dbc_card(card_content, color = "warning", inverse = true)),
            dbc_col(dbc_card(card_content, color = "danger", inverse = true)),
        ],
        className = "mb-4",
    ),
    dbc_row([
        dbc_col(dbc_card(card_content, color = "light")),
        dbc_col(dbc_card(card_content, color = "dark", inverse = true)),
    ]),
]);
